docs/swap_unbiasing.md

```diff
@@ -321,5 +321,12 @@ As mentioned earlier, there is a maximum reachable in the Character Error Rate w
 This maximum is directly due to the constraint of preventing swapping the current character if it has already been swapped.
 
 
+## Taking into account repetitions of characters in natural language
+
+In natural language, our assumption that two consecutive characters are always different is not always true.
+To take this into account, we have to introduce a small multiplicative coefficient to the probability $p$.
+
+For english language, we computed this correction factor to be approximately 1.0655, which is the default in `textnoisr`.
```

Comment:
nit:You may want to explain how to compute this coefficient for other languages. Also it's _English_, with a capital letter.